docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+
+        ![Knative Instance](img/knative_instance.png)
+
+        ![Service Mesh Instance](img/servicemesh_instance.png)
+
+2. Go to the OpenShift AI dashboard and expand **Settings** and select **Serving Runtimes**. You should now see that Single-model serving enabled at the top of the page.
+
+    **NOTE:** *You might need to refresh the page and it could take a few minutes for the changes to be applied.*
+
+    ![Single serving enabled](img/rhoai_single_serving_enabled.png)
+
+### Add a Custom Serving Runtime
+We'll now add a custom serving runtime so we can deploy GGUF versions of models.
+
+1. Click on the **Add serving runtime** button. 
+
+    ![Add Serving runtime button](img/rhoai_add_serving_runtime.png)
+
+2. Select **Single-model serving platform** for the runtime and select **REST** for the API protocol. Upload the *./components/custom-model-serving-runtime/llamacpp-runtime-custom.yaml* file as the serving runtime. Click the **Create** button.
+
+    ![Add Serving runtime](img/rhoai_add_serving_runtime_custom.png)
+
+    **NOTE:** *I've included a pre-built image that is public. You can build your own image with the Containerfile under ./components/ucstom-model-serving-runtime if you would rather pull from your own repository.*
+
+3. If the serving runtime was added was succesfully you should now see it in the list of serving runtimes available. 
+
+    ![Serving runtime list](img/rhoai-llamacpp-runtime.png)
+
+4. Go to your *podman-ai-lab-rag-project* and select **Models**. You should see two model serving type options. Click on the **Deploy model** under the Single-model serving platform. 
+
+    ![Serving runtime list](img/rhoai_project_model_servers.png)
+
+5. Fill in the following values and click the **Deploy** button at the bottom of the form.
+
+      - Model name = **mistral7b**
+      - Serving runtime = **LlamaCPP**
+      - Model framework = **any**
+      - Model server size = **Medium**
+      - Select **New data connection**
+      - Name = **models**
+      - Access key = **minio**
+      - Secret key = **minio123**
+      - Endpoint = **Your Minio API URL**
+      - Region = **us-east-1**
+      - Bucket = **models**
+      - Path = **mistral7b**
+
+    ![Model Deployment 1](img/rhoai_model_serving_1.png)
+
+    ![Model Deployment 2](img/rhoai_model_serving_2.png)
+
+    ![Model Deployment 3](img/rhoai_model_serving_3.png)
+
+6. If your model deploys successfully you should see the following page.
+
+    ![Deployed Model](img/rhoai_deployed_model.png)
+
+7. Test your model to make sure you can send in a request and get a response. You can use the client code that is provided by the model service in Podman AI Lab. 
+
+      Make sure to update the URL in the cURL command to the Inference endpoint on OpenShift AI.
+
+
+      ```
+      curl -k --location 'https://YOUR-OPENSHIFT-AI-INFERENCE-ENDPOINT/v1/chat/completions' --header 'Content-Type: application/json' --data '{
```

Comment:
Why use `-k` if you went through all the process of certificates imports?

Reply:
Removed the -k. It was left in by accident.